roles/os10_users/templates/os10_users.j2

```diff
@@ -13,6 +14,9 @@ os10_users:
     {% if item.username is defined and item.username %}
       {% if item.state is defined and item.state == "absent" %}
 no username {{ item.username }}
+        {% if item.sshkey is defined and item.sshkey %}
+no username {{ item.username }} sshkey
```

Comment:
It may not be necessary here when the user itself is removed.

Reply:
I tested it and it is indeed not necessary